main/guides/orchestration/how-orch-works.md

```diff
@@ -0,0 +1,38 @@
+- **Security:** Ensuring that cross-chain transactions are secure and trustworthy.
+
+To achieve these goals, several foundational technologies and protocols have been developed, with the [Inter-Blockchain Communication (IBC) protocol](https://ibcprotocol.org/) being one of the most prominent.
+
+### Inter-Blockchain Communication (IBC)
+
+The [Inter-Blockchain Communication (IBC) protocol](https://ibcprotocol.org/) can be thought of as the TCP/IP of the blockchain world. Just like TCP/IP allows different computer networks to communicate over the internet, IBC enables different blockchains to communicate with each other.
+
+#### How IBC Works
+
+IBC operates on a simple premise: by adhering to a common set of communication protocols, blockchains can securely transfer data and tokens between one another. Here's a high-level overview of how IBC achieves this:
+
+- **Light Client Verification:** Each blockchain maintains a light client of the other blockchain. A light client is a simplified version of a blockchain client that only retains enough information to verify transactions and proofs, without needing to store the entire blockchain history.
+- **Relayer:** A [relayer](https://medium.com/@cosmos/relayer-in-blockchain-ecosystem-5f8b5b9b8b5e) is an off-chain process or entity that listens for transactions on one blockchain and relays them to another. The relayer facilitates communication between blockchains but does not have any special permissions or abilities; it merely passes messages between chains.
+- **IBC Handshake:** Before two blockchains can communicate, they must perform a handshake, establishing a trusted channel between them. This involves verifying each other's consensus state through light clients.
+- **Packet Transfer:** Once the connection is established, IBC allows the transfer of packets of data. These packets can represent tokens, smart contract commands, or any other type of data. The receiving blockchain uses the light client verification to ensure that the data packet is valid and hasn't been tampered with.
+- **Finality and Acknowledgement:** After a packet is successfully received and processed by the receiving blockchain, an acknowledgment is sent back to the originating blockchain, confirming the completion of the transaction.
+
+The IBC protocol is highly modular, meaning it can be extended and adapted for various use cases and blockchain types. This modularity is what makes IBC a powerful tool for blockchain orchestration, allowing diverse blockchains to participate in a common ecosystem.
+
+### Interchain Accounts (ICA)
+
+[Interchain Accounts (ICA)](https://cosmos.network/learn/interchain-accounts) is a feature built on top of IBC that allows one blockchain to control an account on another blockchain. This is akin to having a remote account that can be operated from a different blockchain. ICA is a critical component of blockchain orchestration, as it enables seamless cross-chain operations and automation.
```

Comment:
This link https://cosmos.network/learn/interchain-accounts is also broken for me. 
I would recommend https://github.com/cosmos/ibc/blob/main/spec/app/ics-027-interchain-accounts/README.md but you may pick something else.